server/app/services/AlertSettings.java

```diff
@@ -23,26 +23,59 @@ public record AlertSettings(
     Boolean isSlim) {
 
   public static AlertSettings empty() {
-    return new AlertSettings(false, Optional.empty(), "", AlertType.NONE);
+    return AlertSettings.builder().build();
   }
 
-  public AlertSettings(Boolean show, Optional<String> title, String text, AlertType alertType) {
-    this(show, title, text, alertType, ImmutableList.of(), /* isSlim= */ false);
+  public static AlertSettingsBuilder builder() {
+    return new AlertSettingsBuilder();
   }
 
-  public AlertSettings(
-      Boolean show,
-      Optional<String> title,
-      String text,
-      AlertType alertType,
-      ImmutableList<String> additionalText,
-      Boolean isSlim) {
-    this(show, title, text, /* unescapedDescription= */ true, alertType, additionalText, isSlim);
-  }
+  public static class AlertSettingsBuilder {
+    private Boolean show = false;
+    private Optional<String> title = Optional.empty();
+    private String text = "";
+    private Boolean unescapedDescription = false;
```

Comment:
suggestion (blocking): I think we want `unescapedDescription` to default to `true` to mimic the past behavior.